content/en/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -84,6 +84,8 @@ object.
 
 ## {{% heading "whatsnext" %}}
 
-* Learn more about [Kubernetes finalizers](/docs/concepts/overview/working-with-objects/finalizers/).
-* Learn about [garbage collection](/docs/concepts/architecture/garbage-collection).
-* Read the API reference for [object metadata](/docs/reference/kubernetes-api/common-definitions/object-meta/#System).
\ No newline at end of file
+Learn more about the following:
+* [Kubernetes finalizers](/docs/concepts/overview/working-with-objects/finalizers/).
+* [garbage collection](/docs/concepts/architecture/garbage-collection).
+* API reference for [object metadata](/docs/reference/kubernetes-api/common-definitions/object-meta/#System).
+* Objets such as [ReplicaSets](/docs/concepts/workloads/controllers/replicaset/), [DaemonSets](/docs/concepts/workloads/controllers/daemonset/), [Deployments](/docs/concepts/workloads/controllers/deployment/), [Jobs](/docs/concepts/workloads/controllers/job/), [CronJobs](/docs/concepts/workloads/controllers/cron-jobs/) and [ReplicationControllers](/docs/concepts/workloads/controllers/replicationcontroller/). 
```

Comment:
Revert this change please, the original syntax is more readable

Reply:
Hi @shannonxtreme,
Here i added one extra point "line number 91" which were mention in page but no reference provided. 
Do you think it is not required to mention? Also advantage of this new syntax is avoiding the phrasing Learn/Read more about XYZ especially with the new addition.
WDYT? Please share your opinion.
 
**Preview Page**: https://deploy-preview-38227--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/working-with-objects/owners-dependents/#what-s-next

Reply:
I'd say revert it, I don't see a value add in this specific change @T-Lakshmi (including the new line). I understand your justification for the updated syntax, but IMO the What's Next section isn't like other lists - each line is independent of the others and are purely interest-based suggestions. the existing format gives us a lot more flexibility with wording as well.

Reply:
Hi @shannonxtreme,
Thanks for the suggestion. I reverted the changes, could you please take an another look!